submissions/lalalilulu/dom-api-menu/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Меню</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="script.js" defer></script>
```

Comment:
It's better to move this `script` to the bottom of the `body` tag

Reply:
I'm using the defer attribute - The defer attribute tells the browser to keep processing the page and load the script in the background, and then run that script when it's loaded. this is considered a good decision (https://learn.javascript.ru/script-async-defer)